app/admin/participant.rb

```diff
@@ -31,6 +31,34 @@
       row :created_at
       row :updated_at
     end
+
+
+    table_for Datastream.where(participant_id: participant.id) do
+      column 'Datasource', :datasource_id do |ds|
+        Datasource.find_by_id(ds).display_name
+      end
+
+      column 'Data points', :id do |dsid|
+        Datapoint.where(datastream_id: dsid).count
+      end
+      column 'Data points (current day)', :id do |dsid|
+        Datapoint.where(datastream_id: dsid, timestamp: (Time.now.at_beginning_of_day)..(Time.now.at_end_of_day)).count
```

Comment:
__[rubocop]__
* Line is too long. [119/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)